File: Python PowerPoint Examples/ShapesApiDemo/deleteSlideTextRanges.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.shapes_api.ShapesApi(configuration)

fileName = "DeleteSlideTextRanges.pptx"
password = ""
storage = ""
inputFolder = "input"
slide_index = 0
shape_index = "0"
paragraph_index = 0
portions = "[0,1]"
result =api.delete_slide_text_ranges(fileName, slide_index=slide_index, shape_index=shape_index, paragraph_index=paragraph_index, portions=portions,password=password, storage=storage,folder=inputFolder)

File: PHP PowerPoint Examples/ShapesApiDemo/deleteSlideTextRanges.php
<?php

use Spire\Cloud\Powerpoint\Sdk\Api\ShapesApi;
use Spire\Cloud\Powerpoint\Sdk\Configuration;

$appId = "your id";
$appKey = "your key";
$baseUrl="https://api.e-iceblue.cn";
$configuration = new Configuration($appId, $appKey,$baseUrl);
$apiInstance = new ShapesApi($configuration);

$name = "DeleteSlideTextRanges.pptx";
$password = null;
$folder = "input";
$storage = null;

$slideIndex = 0;
$shapeIndex = "0";
$paragraphIndex = 0;
$positions = "0,1";
$apiInstance->deleteSlideTextRanges($name, $slideIndex, $shapeIndex, $paragraphIndex, $positions, $password, $folder, $storage);